backend/src/managers/UserManager.ts

```diff
@@ -18,6 +18,7 @@ import {FeedSorting} from '../api/types/entities/common';
 import TheParser from '../parser/TheParser';
 import {UserCache} from './UserCache';
 import {aesDecryptFromBase64, aesEncryptToBase64} from '../parser/CryptoUtils';
+import moment from 'moment';
```

Comment:
Moment.js is a transitive dependency. Either add it explicitly as a dependency into `package.json`, or, better yet, just implement it as a formula, like `Math.abs(date2 - date1) / (3600 * 24 * 1000)`.

Reply:
Fixed, calculating days directly, without the lib